mlrun/datastore/targets.py

```diff
@@ -725,17 +724,7 @@ def __init__(
             after_step = after_step or after_state
 
         if partitioned is None:
-            if all(
-                value is None
-                for value in [
-                    key_bucketing_number,
-                    partition_cols,
-                    time_partitioning_granularity,
-                ]
-            ):
-                partitioned = False
-            else:
-                partitioned = True
+            partitioned = True
```

Comment:
Now I'm a bit concerned - what happens if we create this without all these parameters, and none of them is set later? Will partitioning even work? Will things explode?

Reply:
Yes, it will use default partitioning, same as when you pass `partitioned=True`.